src/columnstore/columnstore_table.hpp

```diff
@@ -16,7 +16,7 @@ class ColumnstoreTable : public TableCatalogEntry {
 public:
     ColumnstoreTable(Catalog &catalog, SchemaCatalogEntry &schema, CreateTableInfo &info, Oid oid, Snapshot snapshot);
 
-    ~ColumnstoreTable();
+    ~ColumnstoreTable() override;
```

Comment:
Otherwise dtor won't be added to virtual table and will be missed.

Reply:
I believe it's still added to vtable
> if a base class declares its destructor virtual, the derived destructor always overrides it
https://en.cppreference.com/w/cpp/language/virtual

But yea, good to have override